pom.xml

```diff
@@ -2579,6 +2579,14 @@
                             <requirePropertyDiverges>
                                 <property>project.description</property>
                             </requirePropertyDiverges>
+                            <requireJavaVendor>
+                                <message>Trino requires Temurin or Oracle JDK for development. Other vendors are not recommended due to lack of testing coverage.</message>
```

Comment:
it is "recommended" not required IMO.

People can develop on whatever they want - it'll get tested using Temurin on CI anyway and that's the quality gate.

Reply:
People were reporting some local issues and the reason was always either Corretto or Azul.

Reply:
I've discussed that in the past and I believe this is right thing to do.

Reply:
It seems reasonable if there are weird issues that waste people's time. Better to be strict and fail up front.

Reply:
Agreed .. especially also since Temurin is free and open source...